backend/go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/implocell/solid-stream
 
 go 1.19
 
-require github.com/gorilla/websocket v1.5.0 // indirect
+require github.com/gorilla/websocket v1.5.0
```

Comment:
🤷

Reply:
Seems correct, should have runned go mod tidy on commit